client/src/CreateBitButton.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import { Button } from 'react-bootstrap';
+import { connect } from 'react-redux';
+import * as actionCreators from './action_creators';
+import CreateBitModal from './CreateBitModal';
+
+class CreateBitButton extends Component {
+
+  constructor(props, context) {
+    super(props, context);
+
+    this.show = this.show.bind(this);
+    this.hide = this.hide.bind(this);
+
+    this.state = {
+      show: false
+    };
+  }
+
+  show() {
+    this.setState({ show: true });
+  }
+
+  hide() {
+    this.setState({ show: false });
+  }
+
+  render() {
+    const { filtering, createBit } = this.props;
+    return ([
+      <Button bsStyle="danger" onClick={this.show} key="create-bit-button">
```

Comment:
a capitalized HTML tag looks so dumb lol

Reply:
It's not HTML, it's JSX :-D